Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN apt-get -y update && \
     apt-get -y install --no-install-recommends \
         git vim parted \
         quilt coreutils qemu-user-static debootstrap zerofree zip dosfstools \
-        bsdtar libcap2-bin rsync grep udev xz-utils curl xxd file kmod bc\
+        libarchive-tools libcap2-bin rsync grep udev xz-utils curl xxd file kmod bc\
```

Comment:
The dockerfile uses debian buster, which has bsdtar, so you're breaking the docker version with this